packages/text-bitmap/src/BitmapFontLoader.ts

```diff
@@ -41,17 +46,17 @@ export class BitmapFontLoader
 
         const baseUrl = BitmapFontLoader.getBaseUrl(this, resource);
         const data = format.parse(resource.data);
-        const textures = {};
+        const textures: Dict<Texture> = {};
 
         // Handle completed, when the number of textures
         // load is the same number as references in the fnt file
-        const completed = (page) =>
+        const completed = (page: any): void =>
         {
             textures[page.metadata.pageFile] = page.texture;
 
             if (Object.keys(textures).length === data.page.length)
             {
-                resource.bitmapFont = BitmapFont.install(data, textures);
+                (resource as any).bitmapFont = BitmapFont.install(data, textures);
```

Comment:
@ivanpopelyshev or @Zyie how do we mixin properties to loader's Resource?

Reply:
from Ivan: 
> Global mixins #6472 were merged , you can make ILoaderResource global mixin.

Reply:
Awesome, will do